reviews/2022-cdk8s-annual.md

```diff
@@ -0,0 +1,101 @@
+from 3 different users, in 3 different organizations (Palo Alto Networks, GridLine, Cisco). There is an additional [contribution](https://github.com/cdk8s-team/cdk8s-cli/pull/324) 
+that doesn’t show up on devstats, probably because the user isn’t associated with any company.
+
+### How many maintainers do you have, and which organizations are they from?
+
+The project currently has 4 maintainers, all from AWS:
+
+* [@iliapolo](https://github.com/iliapolo)
+* [@vinayak-kukreja](https://github.com/vinayak-kukreja)
+* [@sumupitchayan](https://github.com/sumupitchayan)
+* [@ryparker](https://github.com/ryparker)
+
+### What do you know about adoption, and how has this changed since your last review / since you joined Sandbox?
+
+We are currently aware of 7 companies who adopted cdk8s into their production workloads, 
+and an additional 9 who are currently evaluating it. We are in the process of asking them 
+to be added to the [ADOPTERS](https://github.com/cdk8s-team/cdk8s/blob/master/ADOPTERS.md) file.
+
+We are also seeing signs of increasing adoption by observing [NPM download stats](https://npm-stat.com/charts.html?package=cdk8s&from=2022-01-01&to=2023-02-23). 
+They show that over the last year, we have grown from ~12k to ~129k downloads a week for the core library. 
+GitHub stars have also increased from 2280 to [3492](https://star-history.com/#cdk8s-team/cdk8s&Date) stars in the course of the last 15 months. 
+
+In addition, the #cdk8s channel in the [cdk.dev](https://cdk.dev/) workspace now contains ~7k members and we 
+see consistent engagement and knowledge sharing. 
+
+### How has the project performed against its goals since the last review?
+
+Our goals for the project since joining sandbox were:
+
+1. Stabilize the entire toolchain of the project. This was important for our customers to be able 
+to adopt the tool in their production pipelines. We have successfully done so and released a major version 
+for all of components of the project: 
+
+    * [cdk8s-cli](https://github.com/cdk8s-team/cdk8s-cli)
+    * [cdk8s-core](https://github.com/cdk8s-team/cdk8s-core)
+    * [cdk8s-plus](https://github.com/cdk8s-team/cdk8s-plus)
+
+2. Add support for Golang. Users have expressed desire to author cdk8s in applications in Golang, 
+it was in-fact our mostly highly voted [issue](https://github.com/cdk8s-team/cdk8s/issues/119) in the repository.
+3. Help customers safe guard their Kubernetes workloads by integrating cdk8s into policy compliant pipelines 
+and applying policy-as-code tools on the manifests generated by cdk8s. To that end, we created a spec and 
+a pluggable mechanism in the cdk8s-cli that allows third-party vendors to implement plugins for their 
+tools (e.g Kyverno, Checkov, Kubescape, ...). We have partnered with [Datree](https://www.datree.io/) to create a reference 
+implementation of one such plugin, available [here](https://github.com/datreeio/datree-cdk8s).
+
+Official announcements have been made via the AWS blog:
```

Comment:
@iliapolo are there any vendor neutral blog sources you can share?

Reply:
You mean blogs posted by maintainers in other sources? All our blogs currently use the AWS blog. There are many third-party blogs about cdk8s posted on other sources though - would you like links to those?